stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/utils/UiUtils.kt

```diff
@@ -6,6 +6,14 @@ import io.getstream.chat.android.ui.R
 @SuppressWarnings("unused")
 internal object UiUtils {
 
+    private val reactionTypes: Map<String, Int> = mapOf(
+        ReactionType.LOVE.toString() to R.drawable.stream_ic_reaction_love,
```

Comment:
The resource IDs could be a property of `ReactionType` too, instead of being set up here